security/webauthn/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>security-webauthn</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>Quarkus QE TS: Security: WebAuth</name>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-reactive</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-reactive-mysql-client</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-security-webauthn</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-hibernate-reactive-panache</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-junit5</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus.qe</groupId>
+            <artifactId>quarkus-test-service-database</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.dataformat</groupId>
+            <artifactId>jackson-dataformat-cbor</artifactId>
+            <version>2.15.2</version>
```

Comment:
Code linked where is using that library: 
https://github.com/quarkus-qe/quarkus-test-suite/blob/0e3ca4bd996a2f2cc62263bb4e2c75e97b6ae68b/security/webauthn/src/test/java/MyWebAuthnHardware.java#L49

https://github.com/quarkus-qe/quarkus-test-suite/blob/0e3ca4bd996a2f2cc62263bb4e2c75e97b6ae68b/security/webauthn/src/test/java/MyWebAuthnHardware.java#L100
Well, I conducted some tests without utilizing this library and encountered encoded JSON errors. However, it might be possible to leverage an inner Jackson dependency within Vert.x.

Reply:
Makes sense. This code originally comes from `quarkus-test-security-webauthn` that uses same dependency. The `quarkus-test-security-webauthn` is managed by Quarkus BOM, so can we include `quarkus-test-security-webauthn` with test scope instead and bring it as transitive dependency? Please feel free to object.

Reply:
Which reminds me, `<scope>test</scope>` please.

Reply:
Well, I removed quarkus-test-security-webauthn dependency in my "draft" PR to try to test it as close to the client approach as possible, but even so, it was necessary to introduce some classes to make testing possible.
scope testing added.
If I include that library again, I will need to do extra work doing a refactor and delete the classes that have already created but I can do it if it is a better approach. so WDYT?

Reply:
> Well, I removed quarkus-test-security-webauthn dependency in my "draft" PR to try to test it as close to the client approach as possible

Adding `quarkus-test-security-webauthn` in test scope to use its dependencies doesn't affect prod-first approach because it doesn't change a **single** file.

**You don't need to add quarkust-test-security-webauth** because Jackson BOM is part of Quarkus BOM https://github.com/quarkusio/quarkus/blob/main/bom/application/pom.xml#L281 therefore `<version>2.15.2</version>` is absolutely unnecessary and it well lead to our CI running without any point. Drop `<version>2.15.2</version>` and forget about `quarkus-test-security-webauth`.

> If I include that library again, I will need to do extra work doing a refactor and delete the classes that have already created but I can do it if it is a better approach. so WDYT?

I've never suggested you use it, I suggested you include it for transitive deps, which as I mentioned above is also unnecessary. It wouldn't lead to refactoring anyway.

Reply:
ok get it, thanks for the explanation, I will keep the dependency: 
`<dependency>
            <groupId>com.fasterxml.jackson.dataformat</groupId>
            <artifactId>jackson-dataformat-cbor</artifactId>
            <scope>test</scope>
        </dependency>`